data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsService.java

```diff
@@ -203,13 +213,11 @@ private String getS3PathPrefix() {
         return s3PathPrefix;
     }
 
-    private Map<String, Map<String, String>> getColumnDataTypeMap(final SchemaManager schemaManager) {
+    private Map<String, Map<String, String>> getColumnDataTypeMap(final MySqlSchemaManager schemaManager) {
```

Comment:
Can this still be interface SchemaManager ?

Reply:
Yes, it should be possible. Let me make some change after Postgres data type mapping work. It's currently only needed for MySQL.